test/TestCases/VctrBaseMemberFunctions.cpp

```diff
@@ -247,3 +247,19 @@ TEST_CASE ("forEach", "[VctrBaseMemberFunctions]")
 
     REQUIRE (sum1 == sum2);
 }
+
+TEST_CASE ("sizeInBytes", "[VctrBaseMemberFunctions]")
+{
+    auto v = UnitTestValues<int32_t>::template vector<100, 1>();
+    auto a = UnitTestValues<int32_t>::template array<100, 2>();
+    vctr::Span spanFromV (v);
+    vctr::Span spanFromA (a);
+
+    constexpr auto sizeOfA = a.sizeInBytes();
+    constexpr auto sizeOfSpanFromA = spanFromA.sizeInBytes();
+
+    REQUIRE (v.sizeInBytes() == 400);
+    REQUIRE (spanFromV.sizeInBytes() == 400);
+    REQUIRE (sizeOfA == 400);
+    REQUIRE (sizeOfSpanFromA == std::span (a).size_bytes());
```

Comment:
I'm a bit surprised to read `std::span (a)` here. Does this pick the single argument `range` constructor of `std::span`?

Reply:
Good question… I would have expected that it picks the single-argument `array` constructor? I just wanted to check against `std::span::size_bytes()`, which is seems to be implemented only there.